src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -105,7 +105,7 @@ export class NfsFormComponent extends CdForm implements OnInit {
       this.nfsService.filesystems()
     ];
 
-    if (this.router.url.startsWith('/nfs/edit')) {
+    if (this.router.url.startsWith('/file/nfs/edit')) {
```

Comment:
this could be either `/file/nfs/edit` or `/rgw/nfs/edit` according to the recent changes